index.html

```diff
@@ -354,6 +370,58 @@ <h2>
             </tbody>
           </table>
         </section>
+        <section>
+          <h2>
+            <code>CursorCaptureConstraint</code>
+          </h2>
+          <p>
+            The <code><a>CursorCaptureConstraint</a></code> enumerates the conditions 
+            under which the cursor is captured.
+          </p>
+          <pre class="idl">enum CursorCaptureConstraint {
+    "never",
+    "always",
+    "motion"
+};</pre>
+          <table data-link-for="CursorCaptureConstraint" data-dfn-for="CursorCaptureConstraint"
+          class="simple">
+            <tbody>
+              <tr>
+                <th colspan="2">
+                  Enumeration description
+                </th>
+              </tr>
+              <tr>
+                <td>
+                  <dfn><code id="idl-def-CursorCaptureConstraint.never">never</code></dfn>
+                </td>
+                <td>
+                  a <a>never</a> cursor capture constraint omits the cursor from the captured display surface.
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  <dfn><code id="idl-def-CursorCaptureConstraint.always">always</code></dfn>
+                </td>
+                <td>
+                  a <a>always</a> cursor capture constraint includes the cursor in the captured display surface.
```

Comment:
a -> an

Reply:
@juberti my bad .. I will get this fixed.